src/Splunk.OpenTelemetry.AutoInstrumentation/nuget-readme.md

```diff
@@ -0,0 +1,82 @@
+# Using the Splunk.OpenTelemetry.AutoInstrumentation NuGet package
+
+## When to use the NuGet package
+
+Use the NuGet package in the following scenarios:
+
+1. Simplify deployment. For example, a container running a single application.
```

Comment:
In general, I like the idea to have 1. for all points, but this document can be displayed also on nuget.org, Visual Studio, possibly other nuget managers/viewers, and also in readme after upacking nuget. Please make it readable without special parsing. In this case use real order.

Reply:
For reference https://learn.microsoft.com/en-us/nuget/nuget-org/package-readme-on-nuget-org#supported-markdown-features

Reply:
Just to be clear: it will be good if the tool has some markdown lint targeted to the features supported on nuget.org - the repo already has markdown lint.